makefiles/libc/newlib.mk

```diff
@@ -13,7 +13,12 @@ ifeq (1,$(USE_NEWLIB_NANO))
   export LINKFLAGS += -specs=nano.specs
 endif
 
-export LINKFLAGS += -lc -lnosys
+
+ifeq ($(TARGET_ARCH),mips-mti-elf)
+  export LINKFLAGS += -lc
+else
+  export LINKFLAGS += -lc -lnosys
```

Comment:
I think -lnosys might be unused on all platforms now with proper syscall implementations. Could you try unconditionally removing -lnosys and see what Murdock says?